src/worktree.ts

```diff
@@ -32,7 +32,9 @@ export async function generateWorktree(
 
     if (branchExists) {
       await execute(
-        `git fetch --no-recurse-submodules --depth=1 origin ${action.branch}`,
+        `git fetch --no-recurse-submodules --depth=1 ${
+          action.isCrossRepositoryDeployment ? action.repositoryPath : 'origin'
```

Comment:
Haven't we set `origin` to `action.repositoryPath` in `init` now?

Reply:
Yep, I'm realizing now that this bug actually doesn't exist, and was likely caused by some weird merge conflict resolution I created.

I was comparing the logs between the beta build I created and `releases/v4`, the first being the release branch, and the second being the beta, I noticed that the origin lines were missing... 😞 

```shell
Checking configuration and starting deployment… 🚦
Deploying using Deploy Token… 🔑
Configuring git…
/usr/bin/git config user.name JamesIves
/usr/bin/git config user.email iam@jamesiv.es
Git configured… 🔧
Starting to commit changes…
/usr/bin/git ls-remote --heads ***github.com/JamesIves/lab.git gh-pages
03f131b626f0f0483281683be3dba39bbb4f49b0	refs/heads/gh-pages
Creating worktree…
/usr/bin/git fetch --no-recurse-submodules --depth=1 origin gh-pages
fatal: couldn't find remote ref gh-pages
Running post deployment cleanup jobs… 🗑️
/usr/bin/git worktree remove github-pages-deploy-action-temp-deployment-folder --force
fatal: 'github-pages-deploy-action-temp-deployment-folder' is not a working tree
Error: The process '/usr/bin/git' failed with exit code 128
```

```shell
Checking configuration and starting deployment… 🚦
Deploying using Deploy Token… 🔑
Configuring git…
/usr/bin/git config user.name JamesIves
/usr/bin/git config user.email iam@jamesiv.es
/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
/usr/bin/git remote rm origin
/usr/bin/git remote add origin ***github.com/JamesIves/lab.git
Git configured… 🔧
Starting to commit changes…
/usr/bin/git ls-remote --heads ***github.com/JamesIves/lab.git gh-pages
03f131b626f0f0483281683be3dba39bbb4f49b0	refs/heads/gh-pages
Creating worktree…
/usr/bin/git fetch --no-recurse-submodules --depth=1 origin gh-pages
From https://github.com/JamesIves/lab
 * branch            gh-pages   -> FETCH_HEAD
 * [new branch]      gh-pages   -> origin/gh-pages
 ```

I nuked the `releases/v4` branch, re-created, and this bug does not occur any longer.

![You saw nothing](https://media3.giphy.com/media/R7m04yMaGWVeE/giphy.gif?cid=ecf05e47qxdn9ub0388chbilkwzn1sajq2v9noh4sdpz4c1q&rid=giphy.gif)